eigrpd/eigrp_vty.c

```diff
@@ -657,40 +661,9 @@ DEFUN (no_eigrp_if_delay,
 	return CMD_SUCCESS;
 }
 
-DEFUN (eigrp_if_bandwidth,
+DEFPY (eigrp_if_bandwidth,
        eigrp_if_bandwidth_cmd,
-       "eigrp bandwidth (1-10000000)",
-       "EIGRP specific commands\n"
-       "Set bandwidth informational parameter\n"
-       "Bandwidth in kilobits\n")
-{
-	VTY_DECLVAR_CONTEXT(interface, ifp);
-	struct eigrp_interface *ei = ifp->info;
-	uint32_t bandwidth;
-	struct eigrp *eigrp;
-
-	eigrp = eigrp_lookup();
-	if (eigrp == NULL) {
-		vty_out(vty, " EIGRP Routing Process not enabled\n");
-		return CMD_SUCCESS;
-	}
-
-	if (!ei) {
-		vty_out(vty, " EIGRP not configured on this interface\n");
-		return CMD_SUCCESS;
-	}
-
-	bandwidth = atoi(argv[1]->arg);
-
-	ei->params.bandwidth = bandwidth;
-	eigrp_if_reset(ifp);
-
-	return CMD_SUCCESS;
-}
-
-DEFUN (no_eigrp_if_bandwidth,
-       no_eigrp_if_bandwidth_cmd,
-       "no eigrp bandwidth [(1-10000000)]",
+       "[no] eigrp bandwidth (1-10000000)$bw",
```

Comment:
`no eigrp bandwidth` (without a number) doesn't work anymore, is that intentional?